src/providers/sh/commands/deploy.js

```diff
@@ -268,26 +267,13 @@ async function main(ctx) {
     return 0
   }
 
-  let { token } =
-    ctx.authConfig.credentials.find(c => c.provider === 'sh') || {}
 
-  if (!token) {
-    // node file sh [...]
-    const sh = argv[2] === 'sh'
-    const _cmd = `now ${sh ? 'sh ' : ''}login`
-    console.log(error(`You're not logged in! Please run ${cmd(_cmd)}`))
-    return 1
-  }
-
-  const config = ctx.config.sh
+  const {authConfig: { credentials }, config: { sh }} = ctx
+  const {token} = argv.token || credentials.find(item => item.provider === 'sh')
```

Comment:
It looks like prettier is not working here.

Reply:
✅